src/pinefarm/cli/run.py

```diff
@@ -8,24 +8,31 @@
 import rich
 import yaml
 
-from .. import info, install, log, table, tools
+from .. import configs, info, install, log, table, tools
 from ..external import mg5
 from ._base import command
 
 
 @command.command("run")
-@click.argument("dataset")
+@click.argument("pinecard")
 @click.argument("theory-path", type=click.Path(exists=True))
-@click.option("--pdf", default="NNPDF31_nlo_as_0118_luxqed")
+@click.option(
+    "--pdf",
+    help="PDF to compare the original results to the grid",
+    default="NNPDF40MC_nnlo_as_01180_qed",
+)
 @click.option("--dry", is_flag=True, help="Don't execute the underlying code")
-def subcommand(dataset, theory_path, pdf, dry):
-    """Compute a dataset and compare using a given PDF.
+def subcommand(pinecard, theory_path, pdf, dry):
+    """Compute the grids as defined in the given pinecard.
 
-    Given a DATASET name and a THEORY-PATH, a runcard is executed with the
-    suitable external (self-determined).
+    Given a PINECARD and a THEORY-PATH, pinefarm will execute the
+    appropiate external program to generate the grids.
```

Comment:
```suggestion
    appropiate external program to generate the grids.
    
    The given PDF will be used to compare the original results (from the generator) with PineAPPL interpolation - this checks any interpolation issues.
    Setting the DRY flag prevents the generator from actually running.
```
this is user facing, so let's keep it verbose